test/StyleSheet.js

```diff
@@ -1,26 +1,29 @@
-import React from '../src/react-native';
+import { StyleSheet } from '../src/react-native';
 import { expect } from 'chai';
 
-let {StyleSheet} = React;
 
 describe('StyleSheet', () => {
-  let styles = StyleSheet.create({
-    listItem: {
-      flex: 1,
-      fontSize: 16,
-      color: 'white'
-    },
-    selectedListItem: {
-      color: 'green'
-    },
-    headerItem: {
-      fontWeight: 'bold'
-    }
+  let styles;
+
+  beforeEach(function () {
+    styles = StyleSheet.create({
+      listItem: {
+        flex: 1,
+        fontSize: 16,
+        color: 'white'
+      },
+      selectedListItem: {
+        color: 'green'
+      },
+      headerItem: {
+        fontWeight: 'bold'
+      }
```

Comment:
Yes. `comma-dangle` IMO makes the code look weird. I can turn it back on if you'd prefer?